src/views/MainView/MainView.test.tsx

```diff
@@ -0,0 +1,119 @@
+import { render, screen } from '@testing-library/react';
+import { useWeatherApi } from '../../api/hooks/useWeatherApi';
+import { MainView } from './MainView';
+
+jest.mock('../../api/hooks/useWeatherApi');
```

Comment:
Since the hook doesn't have any tests yet, you could instead mock the network request and then you will likely get a lot of it tested for free